index.html

```diff
@@ -26,3 +26,27 @@
 
 </body>
 </html>
+<script>
+//Call this to get input from top input and concat current date seconds to html string
+const inputConcat = () =>{
```

Comment:
Is there a reason you made a separate function to append the string? Combining the inputConcat with the submit handler would save you a DOM query.

Reply:
In retrospect, I usually use the event to reference the source and pass the source to a handler. So I guess this represents a hybrid of that.. albeit sloppy!